rotkehlchen/globaldb/handler.py

```diff
@@ -397,9 +396,9 @@ def _add_underlying_tokens(
                 try:  # underlying token does not exist. Track it
                     asset_id = underlying_token.get_identifier(parent_chain=chain)
                     write_cursor.execute(
-                        """INSERT INTO assets(identifier, type, started, swapped_for)
-                        VALUES(?, ?, ?, ?)""",
-                        (asset_id, AssetType.EVM_TOKEN.serialize_for_db(), None, None),
+                        'INSERT INTO assets(identifier, name, type, started, swapped_for)'
+                        'VALUES(?, ?, ?, ?, ?);',
+                        (asset_id, None, AssetType.EVM_TOKEN.serialize_for_db(), None, None),
```

Comment:
The behavior with the `None` here is the same as before. The reason as for what I understood is because the UnderlyingToken structure doesn't have a name and we provide `None` as default here when adding the token

Reply:
We only add/edit an address, and it may be an eth address of a token without a name (in that part of the if) so then name is indeed none.